postprocessing.py

```diff
@@ -176,12 +198,26 @@ def label_multiclass_image(mask):
     return labeled_image
 
 
+def erode_image(mask, selem_size):
+    selem = rectangle(selem_size, selem_size)
+    eroded_image = binary_erosion(mask, selem=selem)
+    return add_dropped_objects(mask, eroded_image)
+
+
 def dilate_image(mask, selem_size):
     selem = rectangle(selem_size, selem_size)
     return binary_dilation(mask, selem=selem)
 
 
-def dense_crf(img, output_probs, compat_gaussian=3, sxy_gaussian=1, compat_bilateral=10, sxy_bilateral=1, srgb=50):
+def dilate_labeled_image(mask, selem_size):
+    selem = rectangle(selem_size, selem_size)
+    dilated_image = []
+    for category_mask in mask:
+        dilated_image.append(dilation(category_mask, selem=selem))
+    return np.stack(dilated_image)
+
+
+def _dense_crf(img, output_probs, compat_gaussian=3, sxy_gaussian=1, compat_bilateral=10, sxy_bilateral=1, srgb=50):
```

Comment:
@taraspiotr I can definitely see people use this function outside of this module. Any reasons why you want this to be private?